src/component/calculator.js

```diff
@@ -2,6 +2,8 @@
 import React, {useState} from 'react';
 import calculate from '../logic/calculate';
 
+import Quote from './Quote';
+
 // eslint-disable-next-line
```

Comment:
- [ ] it would be better to remove the `Eslint- disable line `  cause Eslint is used to ensure that you write clean code